sortable_listview/templates/sortable_listview/sort_links.html

```diff
@@ -1,15 +1,16 @@
-<p>
-<strong>Sort by: </strong>
-{% for sort_link in sort_link_list %}
-	<span class="sort-link {{ sort_link.attrs }}">
-		<a class="btn btn-primary btn-small" href="{{ sort_link.path }}">
-				<span class="{{ sort_link.indicator }}">
-						{{ sort_link.title }}
-						{% if sort_link.indicator == 'sort-asc' %}▲{% endif %}
-						{% if sort_link.indicator == 'sort-desc' %}▼{% endif %}
-				</span>
-		</a>
-	</span>
-{% endfor %}
-</p>
+{% load i18n %}
 
+<p class="sortable-links">
+    <strong class="sortable-label">{% trans 'Sort by' %}: </strong>
```

Comment:
Do you still want this to be strong? Might make sense to change it to a span and then style it with CSS.